js/components/portal-dashboard/response-details/response-details.tsx

```diff
@@ -25,12 +25,14 @@ interface IProps {
   currentQuestion?: Map<string, any>;
   currentStudentId: string | null;
   hasTeacherEdition: boolean;
+  inQuestionMode: boolean;
   isAnonymous: boolean;
   questions?: Map<string, any>;
   setAnonymous: (value: boolean) => void;
   setCurrentActivity: (activityId: string) => void;
   setCurrentQuestion: (questionId: string) => void;
   setCurrentStudent: (studentId: string | null) => void;
+  setListViewMode: (mode: boolean) => void;
```

Comment:
See my comments below about naming this `toggleQuestionMode` - unless this somehow clashes with upcoming feedback work.

Reply:
Per my comment below, I think the best approach is to change this to:
`setListViewMode: (mode: ListViewMode) => void;`